superset/errors.py

```diff
@@ -100,6 +103,15 @@ class SupersetErrorType(str, Enum):
             ),
         },
     ],
+    SupersetErrorType.MISSING_TEMPLATE_PARAMS_ERROR: [
```

Comment:
wanna update the docs file with the issue codes too?

Reply:
Will do, I wasn't aware we had a doc describing them.

Reply:
If you wanted to add a comment in this file with those details too, that would be much appreciated!

It's the `issue_codes.mdx` file that should be updated

Reply:
Will do, thanks! :)